pixi.toml

```diff
@@ -0,0 +1,49 @@
+[workspace]
+name = "scipystack"
+channels = ["https://prefix.dev/conda-forge"]
+platforms = ["osx-arm64", "linux-64", "win-64"]
+
+
+[dependencies]
+gh = ">=2.79.0,<3"
+
+
+[tasks.clean]
+args = ["repo"]
+cmd = "rm -rf {{ repo }}/{{ repo }}"
+description = "Clean the {repo}/{repo} directory."
+
+[tasks.clone]
+args = ["org", "repo"]
+depends-on = [{ task = "clean", args = ["{{ repo }}"] }]
+cmd = [
+  "cd {{ repo }}",
+  "gh repo clone {{ org }}/{{ repo }}",
```

Comment:
Does this clone with `git:` or `https:` protocol? I think you'd want the former if you have 2FA set up. Not entirely sure, just from memory that is why I have to default to `git`